app/policies/case/sar_policy.rb

```diff
@@ -74,6 +74,20 @@ def progress_for_clearance?
       check_user_is_a_responder_for_case
   end
 
+  def request_amends?
```

Comment:
With this here, I think we still have `Case::BasePolicy#request_amends?`. As that's clearly about FOI cases now, I think we should move that to `Case::FOI::StandardPolicy#request_amends?` now.